packages/graphiql/cypress/e2e/incremental-delivery.cy.ts

```diff
@@ -164,7 +161,6 @@ describeOrSkip('IncrementalDelivery support via fetcher', () => {
             age: 1000,
           },
         },
-        hasNext: false,
```

Comment:
only this test now fails

### Expected

<img width="224" alt="image" src="https://github.com/user-attachments/assets/4f19a963-8183-4c01-b940-4bfcf38b8b44">

### Received

<img width="141" alt="image" src="https://github.com/user-attachments/assets/73ba3fb7-1779-4c80-9705-3d702342f290">

Reply:
ff8fdddfad9289b08ab61df4f88378e207e9e731

Reply:
hmm the patch-package works for me locally, but not for the unit tests... hopefully we can get another alpha released. canary builds on graphql-js appear broken

Reply:
seems didn't help, still have same result 
<img width="152" alt="image" src="https://github.com/user-attachments/assets/6dbd402b-c262-4ee7-ab61-3128296119c7">

Reply:
had the patch in the wrong folder, now it's working in our tests, although any servers using alpha.4 or alpha.5 are going to have the bug unless they also patch :(